main.js

```diff
@@ -87,7 +87,7 @@ function updateLatestGuess() {
   gameCount.push("+"); // twoGuess count
   submitButton.disabled = true;
   clearButton.disabled = true;
-  resetButton.disabled = true;
+  // resetButton.disabled = true; //reset button remain active becuase it is used to reset game at any time
```

Comment:
If you haven't entered anything into any of the fields, do you really need to reset the game?